### ISSUE TITLE
[NetworkUpsTools] Binding throws errors if host unreachable

### ISSUE THREAD

**Issue by udo1toni** [openhab/openhab1-addons#5430]
## Expected Behavior
The binding should only tell that connection timed out and/or that the host is unreachable (one line in openhab.log)

At least these two errors are very common and should be covered by the binding, as they appear once a minute (default refresh rate of binding, changing this parameter would affect the binding at all)

## Current Behavior

Instead of one line, there is much additional garbage
```
2018-01-07 01:09:26.596 [ERROR] [ools.internal.NetworkUpsToolsBinding] - Nut processing error
java.net.ConnectException: Die Wartezeit für die Verbindung ist abgelaufen (Connection timed out)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) [?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) [?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) [?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) [?:?]
        at java.net.Socket.connect(Socket.java:589) [?:?]
        at java.net.Socket.connect(Socket.java:538) [?:?]
        at java.net.Socket.<init>(Socket.java:434) [?:?]
        at java.net.Socket.<init>(Socket.java:211) [?:?]
        at org.networkupstools.jnut.StringLineSocket.connect(StringLineSocket.java:77) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.StringLineSocket.<init>(StringLineSocket.java:66) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.connect(Client.java:208) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.connect(Client.java:182) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.<init>(Client.java:165) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.openhab.binding.networkupstools.internal.NetworkUpsToolsBinding.execute(NetworkUpsToolsBinding.java:98) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.openhab.core.binding.AbstractActiveBinding$BindingActiveService.execute(AbstractActiveBinding.java:144) [200:org.openhab.core.compat1x:2.3.0.201712221545]
        at org.openhab.core.service.AbstractActiveService$RefreshThread.run(AbstractActiveService.java:166) [200:org.openhab.core.compat1x:2.3.0.201712221545]

2018-01-07 20:37:35.044 [ERROR] [ools.internal.NetworkUpsToolsBinding] - Nut processing error
java.net.NoRouteToHostException: Keine Route zum Zielrechner (Host unreachable)
        at java.net.PlainSocketImpl.socketConnect(Native Method) ~[?:?]
        at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350) [?:?]
        at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206) [?:?]
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188) [?:?]
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) [?:?]
        at java.net.Socket.connect(Socket.java:589) [?:?]
        at java.net.Socket.connect(Socket.java:538) [?:?]
        at java.net.Socket.<init>(Socket.java:434) [?:?]
        at java.net.Socket.<init>(Socket.java:211) [?:?]
        at org.networkupstools.jnut.StringLineSocket.connect(StringLineSocket.java:77) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.StringLineSocket.<init>(StringLineSocket.java:66) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.connect(Client.java:208) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.connect(Client.java:182) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.networkupstools.jnut.Client.<init>(Client.java:165) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.openhab.binding.networkupstools.internal.NetworkUpsToolsBinding.execute(NetworkUpsToolsBinding.java:98) [232:org.openhab.binding.networkupstools:1.12.0.201712210210]
        at org.openhab.core.binding.AbstractActiveBinding$BindingActiveService.execute(AbstractActiveBinding.java:144) [200:org.openhab.core.compat1x:2.3.0.201712221545]
        at org.openhab.core.service.AbstractActiveService$RefreshThread.run(AbstractActiveService.java:166) [200:org.openhab.core.compat1x:2.3.0.201712221545]
```

**Comment by 9037568** [comment]
First, I've reduced the severity of these messages to WARN instead of ERROR. Second, I reorganized the catches and rewrote the logging statements to emit only the message from the exception instead of a stacktrace.

[Here's a test jar](https://drive.google.com/open?id=1xadM7a82Hw0luIfoGGbfiUK2tIN7grHs).

Code change [is here](https://github.com/openhab/openhab1-addons/compare/master...9037568:nups?expand=1).
